src/concrete/ml/onnx/convert.py

```diff
@@ -158,6 +158,7 @@ def get_equivalent_numpy_forward_from_torch(
 
 def get_equivalent_numpy_forward_from_onnx(
     onnx_model: onnx.ModelProto,
+    lsbs_to_remove: List[int] = None,
```

Comment:
typing should be `Optional[List[int]`

Reply:
I tried it, but mypy is yelling.

lsbs_to_remove will disappear when we add truncate.